auacm/app/modules/competition_manager/views.py

```diff
@@ -1,7 +1,6 @@
 from flask import request
 from flask.ext.login import current_user, login_required
 from app import app
-from app.database import session
```

Comment:
What's the motivation to move away from `from app.database import session` to `import app.database as database` then accessing `session` by `database.session`?